components/CrosswordLayout/CrosswordLayout.test.js

```diff
@@ -1,63 +1,95 @@
+  const currentClueId = 1;
+  const entryCharacterLength = 5;
+
   render(
-    <CrosswordLayout onCurrentClueId={"1"} crosswordClues={crosswordClues} />
+    <CrosswordLayout
+      onCurrentClueId={onCurrentClueId}
+      crosswordClues={crosswordClues}
+      onData={onData}
+      onChangeData={onChangeData}
+      currentClueId={currentClueId}
+      entryCharacterLength={entryCharacterLength}
+    />
   );
+  const toggleButton = screen.getByRole("button", {
+    name: "Fragen anzeigen",
+  });
+  await userEvent.click(toggleButton);
 
   const questionList = screen.getByRole("list");
   expect(questionList).toBeInTheDocument();
 
   const questionItems = screen.getAllByRole("listitem");
   expect(questionItems).toHaveLength(9);
 
-  const question1 = screen.getByText(
-    "Der Ort, an dem die Seelen der Verstorbenen hingelangen?"
+  const question1 = screen.getByRole("list");
+  expect(question1).toHaveTextContent(
+    /Der Ort, an dem die Seelen der Verstorbenen hingelangen?/i
   );
-  expect(question1).toBeInTheDocument();
 
-  const question2 = screen.getByText("Kleines Nagetier.");
-  expect(question2).toBeInTheDocument();
+  const question2 = screen.getByRole("list");
+  expect(question2).toHaveTextContent(/Kleines Nagetier/i);
 
-  const question3 = screen.getByText("Eines der 4 Elemente.");
-  expect(question3).toBeInTheDocument();
+  const question3 = screen.getByRole("list");
+  expect(question3).toHaveTextContent(/Eines der 4 Elemente/i);
 
-  const question4 = screen.getByText(
-    "Wie nennt man den Verfasser eines Buches?"
+  const question4 = screen.getByRole("list");
+  expect(question4).toHaveTextContent(
+    /Wie nennt man den Verfasser eines Buches?/i
   );
-  expect(question4).toBeInTheDocument();
 
-  const question5 = screen.getByText("Größeres Gewässer.");
-  expect(question5).toBeInTheDocument();
+  const question5 = screen.getByRole("list");
+  expect(question5).toHaveTextContent(/Größeres Gewässer/i);
 
-  const question6 = screen.getByText("Eisige Kälte.");
-  expect(question6).toBeInTheDocument();
+  const question6 = screen.getByRole("list");
+  expect(question6).toHaveTextContent(/Eisige Kälte/i);
 
-  const question7 = screen.getByText(
-    "Bezeichnung für eine Person, die du anhimmelst."
+  const question7 = screen.getByRole("list");
+  expect(question7).toHaveTextContent(
+    /Bezeichnung für eine Person, die du anhimmelst/i
   );
-  expect(question7).toBeInTheDocument();
 
-  const question8 = screen.getByText(
-    "Knuspriges Topping auf Kuchen, Muffins oder Eis."
+  const question8 = screen.getByRole("list");
+  expect(question8).toHaveTextContent(
+    /Knuspriges Topping auf Kuchen, Muffins oder Eis/i
   );
-  expect(question8).toBeInTheDocument();
 
-  const question9 = screen.getByText("Scheues Huftier mit Geweih.");
-  expect(question9).toBeInTheDocument();
+  const question9 = screen.getByRole("list");
+  expect(question9).toHaveTextContent(/Scheues Huftier mit Geweih/i);
 });
```

Comment:
crazy testing, good job!

Reply:
I regret that I started it 😮‍💨, but now I'm going through with it too